Sql integration and updates to text that no changes have been made this academic year

MPARMAR08 opened the pull request:
## Pull request overview

Give a general description of why a change is being made, include issue number(s) being fixed if relevant

## Pull request checklist

Please check if your PR fulfils the following:
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been reviewed and added / updated if needed (for bug fixes / features)
- [ ] Tests have been run locally and are passing (`run_tests_locally()`)
- [ ] Code is styled according to tidyverse styling (checked locally with `tidy_code()`)


## What is the current behaviour?

Please describe the current behaviour that you are modifying, or link to a relevant issue. Include screenshots for UI changes where possible.


## What is the new behaviour?

Please describe the behaviour or changes that are being added by this PR. Include screenshots for UI changes where possible.


## Anything else

Add any notes for people reviewing and testing your code that are appropriate. Tag a @person to review if someone in particular needs to see this.

MPARMAR08 committed:
Updates to reflect SQL changes made by Liam earlier this year.
Also updated 'update text' to reflect lack of update this academic year (no prior attainment due to COVID-19).

MPARMAR08 committed:
Added files to datafile log

rmbielby commented on README.md:
```diff
@@ -69,7 +69,7 @@ Around 3-7 days prior to a data update, create a new branch from main and run `r
 2. Open the R project in R Studio.
 
-3. Run `renv::restore()` to install dependencies.
+3. Run `Y` to install dependencies.
```
Think you still want to be running `renv::restore()` here? Just that you can to enter a `Y` to confirm that you want to go ahead with the installations when you do run it?

Maybe worth adding that when making updates, it's worth running `renv::update()` followed by `renv::snapshot()` as well.

MPARMAR08 committed:
Run renv::restore and renv:snapshot
Also run 2 tests:

chromote::local_chrome_version(binary = "chrome-headless-shell", quiet = FALSE)

shinytest2::test_app()

MPARMAR08 committed:
Stylistic changes

rmbielby committed:
Trying to figure out what's going on with progress and hms

rmbielby committed:
Trying to figure out what's going on with progress and hms

rmbielby committed:
Reverting to hms@1.1.3 as GitHub actions can't find 1.1.4
